openg2p_disbursement/models/openg2p_disbursement_batch_transaction.py

```diff
@@ -226,20 +226,20 @@ def create_bulk_transfer(self):
         url = "http://bulk-connector.sandbox.fynarfin.io/batchtransactions"
```

Comment:
please remove this URL from the code and make it configurable or env variable

Reply:
Will add it to environment variable